packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -205,6 +218,14 @@ export interface GatewayRouteSpecConfig {
    * @default - no grpc spec
    */
   readonly grpcSpecConfig?: CfnGatewayRoute.GrpcGatewayRouteProperty;
+
+  /**
+   * The priority for the gateway route. Gateway routes are matched based on the specified
+   * value, where 0 is the highest priority.
+   *
+   * @default - no particular priority
```

Comment:
This is not clear.

Reply:
I can update this comment. This was copied from existing Route comment.